library/src/main/res/values/attrs.xml

```diff
@@ -4,6 +4,9 @@
     <declare-styleable name="AboutLibraries">
         <attr name="about_libraries_window_background" format="color|reference"/>
         <attr name="about_libraries_card" format="color|reference"/>
+        <attr name="about_libraries_title_description" format="color|reference"/>
+        <attr name="about_libraries_text_description" format="color|reference"/>
+        <attr name="about_libraries_dividerDark_description" format="color|reference"/>
```

Comment:
I think this should be just named `about_libraries_divider_description`. What do you think?